Header header
uint8 detection_count
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/Image segmentation_image
DetectionArray[] detection_array

